pkg/get/get_test.go

```diff
@@ -1504,3 +1504,43 @@ func Test_DownloandHelm(t *testing.T) {
 	}
 
 }
+
+func Test_DownloadKubie(t *testing.T) {
+	tools := MakeTools()
+	name := "kubie"
+
+	tool := getTool(name, tools)
+
+	tests := []test{
```

Comment:
Given that there's no 32-bit ARM binary for Raspberry Pi OS, would you mind also raising an issue on https://github.com/sbstp/kubie and requesting it?

Reply:
@alexellis, done asked for a 32bit release

Reply:
kubie released version 0.15 with 32bit support

Reply:
updated version to 0.15.1 with 32bit